### PR TITLE
Deploy Hotfix to Prod - v1.7.2

### PR DESCRIPTION
**Pull request by IS-lamberto**

### CODE REVIEW

File: .drone2.yml [hunk 1]
```diff
@@ -1302,6 +1304,197 @@ steps:
...
+      - pip3 install awscli
+      - mkdir ~/.ssh
+      - echo -e "$GIT_PRIVATE_SSH" > ~/.ssh/id_rsa
+      - chmod 600 ~/.ssh/id_rsa
+      - touch ~/.ssh/known_hosts
+      - chmod 600 ~/.ssh/known_hosts
+      - ssh-keyscan -H github.com > /etc/ssh/ssh_known_hosts 2> /dev/null
+      - mkdir ~/.aws
+      - echo -e "[profile target_acc]\nrole_arn = $SRC_AWS_ACCOUNT_ROLE\ncredential_source = Environment\nrole_session_name = AWSACCOUNT" > ~/.aws/config
+      - LAMBDA_FUNCTION=notify-jaqu-caz-lambda-preprod
```
Comment (dtginformed) [inline .drone2.yml:1375]: do these need an `export` in front of them?

File: .drone2.yml [hunk 2]
```diff
@@ -1302,6 +1304,197 @@ steps:
...
```
Comment (dtginformed) [inline .drone2.yml:1429]: do these need an `export` in front of them?

**Review by dtginformed — CHANGES_REQUESTED** [review]

File: .drone2.yml [hunk 3]
```diff
@@ -1302,6 +1304,197 @@ steps:
...
+      - ENV=preprod
+      - SIGNED_URL=`aws lambda get-function --profile target_acc --function-name $LAMBDA_FUNCTION --qualifier $ENV| jq --raw-output '.Code.Location'`
+      - VERSION=`aws lambda get-function --profile target_acc --function-name $LAMBDA_FUNCTION --qualifier $ENV| jq --raw-output '.Configuration.Version'`
```
Comment (dtginformed) [inline .drone2.yml:1378]: do these need an `export` in front of them?

File: .drone2.yml [hunk 4]
```diff
@@ -1302,6 +1304,197 @@ steps:
...
```
Comment (dtginformed) [inline .drone2.yml:1432]: do these need an `export` in front of them?

**Review by dtginformed — CHANGES_REQUESTED** [review]

**Review by dtginformed — APPROVED** [review]

**Review by kingarj — APPROVED** [review]

**Pull request merged**